src/components/LeftSidebar/ConversationsList/Conversation.vue

```diff
@@ -93,13 +93,27 @@
 				{{ t('spreed', 'Delete conversation') }}
 			</NcActionButton>
 		</template>
+		<template v-else-if="item.token" slot="actions">
+			<NcActionButton icon="icon-clippy"
+				@click.stop.prevent="handleCopyLink">
+				<template #icon>
+					<ArrowRight :size="16" />
+				</template>
+				{{ t('spreed', 'Join conversation') }}
+			</NcActionButton>
+			<NcActionButton icon="icon-clippy"
+				@click.stop.prevent="handleCopyLink">
+				{{ t('spreed', 'Copy link') }}
+			</NcActionButton>
```

Comment:
**issue (blocking):** Is it intended, that these buttons has the same handler `handleCopyLink`?